### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Constructions.lean [hunk 1]
```diff
@@ -13,9 +13,9 @@ import Mathlib.Topology.Homeomorph
 /-!
 # Topological space structure on the opposite monoid and on the units group
 
-In this file we define `topological_space` structure on `Mᵐᵒᵖ`, `Mᵃᵒᵖ`, `Mˣ`, and `add_units M`.
+In this file we define `TopologicalSpace` structure on `Mᵐᵒᵖ`, `Mᵃᵒᵖ`, `Mˣ`, and `add_units M`.
 This file does not import definitions of a topological monoid and/or a continuous multiplicative
-action, so we postpone the proofs of `has_continuous_mul Mᵐᵒᵖ` etc till we have these definitions.
+action, so we postpone the proofs of `HasContinuousMul Mᵐᵒᵖ` etc till we have these definitions.
```
Comment (dupuisf) [inline Mathlib/Topology/Algebra/Constructions.lean:18]: Shouldn't this be `ContinuousMul`?

**Review by dupuisf — COMMENTED** [review]
Thanks!

bors d+

File: Mathlib/Topology/Algebra/Constructions.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Topology/Algebra/Constructions.lean:18]: I'm not sure, it's not ported yet :) I don't think we've been consistent about dropping the `has` prefix. I'll leave it like this for now.

**Pull request closed without merging**